tests/tcp_stream.rs

```diff
@@ -639,14 +639,13 @@ fn tcp_reset_close_event() {
         vec![ExpectEvent::new(ID1, Readiness::READ_CLOSED)],
     );
 
-    // Make sure we quiesce. `expect_no_events` seems to flake sometimes on mac/freebsd.
-    loop {
-        poll.poll(&mut events, Some(Duration::from_millis(100)))
-            .expect("poll failed");
-        if events.iter().count() == 0 {
-            break;
-        }
-    }
+    // Something weird is going on here. Linux keeps emitting HUP, so even though everything is
```

Comment:
I'm assuming this is using the `poll` interface? If so, please remove along with the `force-old-poll` feature.